packages/predicates/src/is-noyes/index.ts

```diff
@@ -87,39 +79,42 @@ export const isNo = isFalse;
  * // true
  *
  * isOff('no');
+ * // false
  */
-export const isOff = isFalse;
+export const isOff = (val: unknown) => !val || test(['off'], val);
```

Comment:
There's a little bit of a discrepancy between `isOff` and `isOn`, where `isOff` will accept any falsy value || `off` variants, instead of just `false` like how `isOn` only accepts `true` || `on` variants

I would expect them to be the same level of rigid/flexible. And I'd expect them both to accept things like `0` or `1` since that's relatively common term for on/off

Reply:
Good point.